zit/test/test_tag_invoke.cpp

```diff
@@ -6,7 +6,8 @@
 #include <type_traits>
 
 inline constexpr struct cpo_t final {
-  inline constexpr auto operator()(auto &&... as) const
+  template <typename... AS>
+  inline constexpr auto operator()(AS &&... as) const
```

Comment:
We should refactor all of these away from decltype now that we don't need it. Also the naming convention for template params is CamelCase not MACRO_CASE.
I'll do that, obviously, as well as reformat with clang-format.